src/hearings/containers/request-hearing/hearing-panel-required/hearing-panel-required.component.ts

```diff
@@ -0,0 +1,71 @@
+import { AfterViewInit, Component, OnDestroy, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { ActivatedRoute } from '@angular/router';
+import { FeatureToggleService } from '@hmcts/rpx-xui-common-lib';
+import { Store } from '@ngrx/store';
+import {
+  ACTION
+} from '../../../models/hearings.enum';
+import { HearingsService } from '../../../services/hearings.service';
+import * as fromHearingStore from '../../../store';
+import { RequestHearingPageFlow } from '../request-hearing.page.flow';
+
+@Component({
+  selector: 'exui-hearing-panel-required',
+  templateUrl: './hearing-panel-required.component.html'
+})
+export class HearingPanelRequiredComponent extends RequestHearingPageFlow implements OnInit, AfterViewInit, OnDestroy {
+  public hearingPanelRequiredForm: FormGroup;
+  public hearingPanelRequired: boolean = false;
+  public validationErrors: { id: string, message: string }[] = [];
+
+  constructor(
+    private readonly formBuilder: FormBuilder,
+    protected readonly hearingStore: Store<fromHearingStore.State>,
+    protected readonly hearingsService: HearingsService,
+    protected readonly featureToggleService: FeatureToggleService,
+    protected readonly route: ActivatedRoute) {
+    super(hearingStore, hearingsService, featureToggleService, route);
+  }
+
+  public ngOnInit(): void {
+    this.hearingPanelRequired = this.hearingRequestMainModel?.hearingDetails?.isAPanelFlag ?? false;
+    this.initForm();
+  }
+
+  public ngAfterViewInit(): void {
+    this.fragmentFocus();
+  }
+
+  public initForm(): void {
+    this.hearingPanelRequiredForm = this.formBuilder.group({
+      hearingPanelRequired: [this.hearingPanelRequired, Validators.required]
+    });
+  }
+
+  public executeAction(action: ACTION): void {
+    if (action === ACTION.CONTINUE) {
+      this.updateHearingConditions();
```

Comment:
It doesn't appear to be the norm to update the hearing store in this component, it would seem to be more normal for this to happen in the page.flow. Does this need to happen here?

Reply:
updating the hearing conditions is only used by components that have conditional routes for example https://github.com/hmcts/rpx-xui-webapp/blob/8302d97b4ea0a057d9c1f245722d6e7d43f8bca2/src/hearings/containers/request-hearing/hearing-venue/hearing-venue.component.ts#L89